internal/pkg/service/common/etcdop/watch.go

```diff
@@ -82,9 +90,8 @@ func (v Prefix) Watch(ctx context.Context, client etcd.Watcher, handleErr func(e
 
 // GetAllAndWatch loads all keys in the prefix by the iterator and then watch for changes.
 // initDone channel signals end of the load phase and start of the watch phase.
-func (v Prefix) GetAllAndWatch(ctx context.Context, client *etcd.Client, handleErr func(err error), opts ...etcd.OpOption) (out <-chan Events, initDone <-chan error) {
+func (v Prefix) GetAllAndWatch(ctx context.Context, client *etcd.Client, handleErr func(err error), opts ...etcd.OpOption) (out <-chan Events) {
 	outCh := make(chan Events)
-	initDoneCh := make(chan error)
```

Comment:
`GetAllAndWatch` mal v return `initDone` channel typu `error`.
- To malo sluzit na to, ze vieme cakat na nacitanie vsetkych hodnot `GetAll`,  `Watch` potom uz bezi v pozadi.
- Takto mozeme vypnut `worker node` s chybou, ak sa pri starte napr. nepodari pripojit do etcd.